Bug: resource_flexibleengine_lts_group - ttl_in_days default value

Hi,

I would like to report bug or lack in official of documentation about the argument `ttl_in_days` in the `resource_flexibleengine_lts_group`

### Terraform Version
```shell
Terraform v1.2.2
on darwin_amd64
+ provider registry.terraform.io/flexibleenginecloud/flexibleengine v1.29.0
```

### Affected Resource(s)
- resource_flexibleengine_lts_group

### Terraform Configuration Files
```hcl
resource "flexibleengine_lts_group" "log_group" {
  region = var.vpc_config.region
  group_name = "${var.vpc_config.name}-log-group"
}
```

### Debug Output
```shell
module.vpc.flexibleengine_lts_group.log_group: Creating...
2022-06-15T14:18:13.921+0200 [ERROR] vertex "module.vpc.flexibleengine_lts_group.log_group" 
error: Error creating log group: Bad request with: [POST https://lts.eu-west-0.prod-cloud-ocb.orange-business.com/v2.0/3ca963528faf4f7c9cc1558a5ea396e8/log-groups], 
error message: {"error_code":"LTS.0106","error_msg":"The log group TTL day must be integer and between 1 and 30"}

Error: Error creating log group: Bad request with: [POST https://lts.eu-west-0.prod-cloud-ocb.orange-business.com/v2.0/3ca963528faf4f7c9cc1558a5ea396e8/log-groups], 
error message: {"error_code":"LTS.0106","error_msg":"The log group TTL day must be integer and between 1 and 30"}
```

### Expected Behavior
Creation of the ressource `flexibleengine_lts_group.log_group`

### Actual Behavior
Error on creating log group, no default value and not documented as mandatory argument

https://registry.terraform.io/providers/FlexibleEngineCloud/flexibleengine/latest/docs/resources/lts_group

### FYI
I try to set `ttl_in_days` as argument but terraform apply throw an error 
```
Error: Value for unconfigurable attribute

Can't configure a value for "ttl_in_days": its value will be decided automatically based on the result of applying this configuration.
```

### Steps to Reproduce
1. Just create a flexibleengine_lts_group resource like
```hcl
resource "flexibleengine_lts_group" "log_group" {
  group_name = "test-log-group"
}
```
2. `terraform apply`

--- a/flexibleengine/resource_flexibleengine_lts_group.go
+++ b/flexibleengine/resource_flexibleengine_lts_group.go
@@ -48,6 +48,7 @@ func resourceLTSGroupV2Create(d *schema.ResourceData, meta interface{}) error {
 
 	createOpts := &loggroups.CreateOpts{
 		LogGroupName: d.Get("group_name").(string),
+		TTL:          7, // fixed to 7days
 	}
 
 	log.Printf("[DEBUG] Create Options: %#v", createOpts)

--- a/go.mod
+++ b/go.mod
@@ -4,7 +4,7 @@ go 1.16
 
 require (
 	github.com/aws/aws-sdk-go v1.25.3
-	github.com/chnsz/golangsdk v0.0.0-20220615020710-87cd9b9a3040
+	github.com/chnsz/golangsdk v0.0.0-20220616110846-5f3079766b1d
 	github.com/hashicorp/errwrap v1.1.0
 	github.com/hashicorp/go-cleanhttp v0.5.2
 	github.com/hashicorp/go-multierror v1.1.1

